### PR TITLE
Add support for using iOSSnapshotTestCase as the snapshot engine when installing via SPM

### PR DESCRIPTION
**Pull request by NickEntin**

### CODE REVIEW

**Comment by NickEntin** [comment]
- [ ] Verify that the ObjC target still works now that it's accessing private methods in a different target.

**Comment by alanzeino** [comment]
Let me know if you need me to do anything upstream in our repo!

File: Sources/AccessibilitySnapshot/iOSSnapshotTestCase/ObjC/include/FBSnapshotTestCase_Accessibility.h [hunk 1]
```diff
@@ -14,7 +14,11 @@
 //  limitations under the License.
 //
 
-#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+//#if SWIFT_PACKAGE
+@import iOSSnapshotTestCaseCore;
+//#else
+//#import <iOSSnapshotTestCaseCore/FBSnapshotTestCase.h>
+//#endif
```
Comment (NickEntin) [inline Sources/AccessibilitySnapshot/iOSSnapshotTestCase/ObjC/include/FBSnapshotTestCase_Accessibility.h:21]: This is the problem. Ideally this should be
```c
#if SWIFT_PACKAGE
@import iOSSnapshotTestCaseCore;
#else
#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
#endif
```
(I think the `@import` should actually be `#import` with a specific file, but that wouldn't compile 🤷‍♂️)

The issue is that `SWIFT_PACKAGE` isn't available in public header files as far as I can tell, so there's no way to distinguish whether it's being built with SPM or CP.

@alanzeino any idea how to work around this?

Comment (NickEntin) [reply Sources/AccessibilitySnapshot/iOSSnapshotTestCase/ObjC/include/FBSnapshotTestCase_Accessibility.h]: Nevermind, turns out this import isn't actually needed at all 🤦‍♂️

**Review by AliceAponasko — APPROVED** [review]

File: Sources/AccessibilitySnapshot/iOSSnapshotTestCase/ObjC/FBSnapshotTestCase_Accessibility.m [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+// This file is intentionally left blank. SPM requires the target to have an implementation file to recognize that it is
+// an Objective-C target.
```
Comment (jhneves) [inline Sources/AccessibilitySnapshot/iOSSnapshotTestCase/ObjC/FBSnapshotTestCase_Accessibility.m:18]: TIL

**Pull request merged**